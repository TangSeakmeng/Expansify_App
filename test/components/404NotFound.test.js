import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../src/componenets/404NotFound';

test('should render 404NotFound page correctly', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();
})
