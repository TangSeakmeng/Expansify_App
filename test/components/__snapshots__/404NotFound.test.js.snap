// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404NotFound page correctly 1`] = `
<div>
  404 Not Found! 
  <[object Object]
    to="/"
  >
    Goto Home Page
  </[object Object]>
</div>
`;
